app/styles/global/_base.scss

```diff
@@ -1,3 +1,8 @@
 // - - - - - - - - - - - - - - - - - - -
 // - - base
 // project defaults for base elements - h1-h6, p, a, etc.
+
+html, body {
+  height:100%;
+  margin:0;
```

Comment:
bootstrap does this in `normalize.scss`. was it being overwritten somewhere else?